addons/website_sale/static/src/js/website_sale.js

```diff
@@ -43,6 +44,15 @@ odoo.define('website_sale.cart', function (require) {
                 }
             }
         }, 1000);
+    }).on("click", function (ev) {
+        if (!popoverRpc || popoverRpc.state() === 'resolved') {
+            return;
+        }
+        ev.preventDefault();
+        clearTimeout(shopping_cart_link_counter);
+        popoverRpc.then(function () {
+            window.location.href = this.href;
```

Comment:
Really strange to use `this` in a new `function`, I am surprised it works. Use `ev.currentTarget`.